nilearn/decoding/__init__.py

```diff
@@ -2,3 +2,4 @@
 """
 
 from .searchlight import SearchLight
```

Comment:
General comment: the name is too general. How about "MetaEstimator" to include both classification and regression.